### PR TITLE
Replace handwritten table with Quasar table

### PR DESCRIPTION
**Pull request by sverhoeven**
Refs #31

Also
- removed badge sorting
- added stdout/stderr in expanding rows

### CODE REVIEW

File: fairtally/data/index.html.template [hunk 1]
```diff
@@ -1,200 +1,153 @@
+<!DOCTYPE html>
 <html>
+<!--
+    WARNING! Make sure that you match all Quasar related
+    tags to the same version! (Below it's "@2.0.0-beta.8")
+  -->
+
+<title>Fairtally results</title>
+
+<head>
```
Comment (eriktks) [inline fairtally/data/index.html.template:10]:
```suggestion
<head>
<title>Fairtally results</title>
```

**Comment by sverhoeven** [comment]
_home_verhoes_git_fair-software_fairtally_bla html

**Review by eriktks — APPROVED** [review]
Checked the changed, installed the PR locally, checked that `pip install .` also updated the file `fairtally/data/index.html.template` in the repo (issue #26), ran a `--html` test, inspected the result and all looks fine. Great work!

**Comment by sverhoeven** [comment]
Thanks for reviewing.

**Comment by jspaaks** [comment]
https://github.com/fair-software/fairtally/pull/32#issuecomment-789790418

so pwetty!

**Pull request merged**